ang/civicase/activity/calendar/directives/activities-calendar.directive.js

```diff
@@ -150,8 +150,22 @@
 
   module.controller('civicaseActivitiesCalendarController', civicaseActivitiesCalendarController);
 
+  /**
+   * Activities Calendar Controller
+   *
+   * @param {*} $q $q
+   * @param {*} $rootScope $rootScope
+   * @param {*} $route $route
+   * @param {*} $sce $sce
+   * @param {*} $scope $scope
+   * @param {*} ContactsCache ContactsCache
+   * @param {*} crmApi crmApi
+   * @param {*} formatActivity formatActivity
+   * @param {*} getActivityFeedUrl getActivityFeedUrl
+   * @param {*} ActivityStatusType ActivityStatusType
```

Comment:
![...](https://i.imgur.com/piGMDmW.gif)

Reply:
Why are we enforcing a standard to then cheat it? 😅

Reply:
well to be honest, I had to write description for almost 40 files. so I found a quick way of fixing it. Now it seems like, it would have been better to just use `--no-verify`, then atleast we could write them properly later :/